platform/macos/src/MGLMapView.h

```diff
@@ -48,6 +48,7 @@ NS_ASSUME_NONNULL_BEGIN
     ensuring that your use adheres to the relevant terms of use.
  */
 IB_DESIGNABLE
+MGL_EXTERN
```

Comment:
I tried this, but couldn't get to a conclusive answer since Xcode's rebuilding is very erratic and it doesn't seem to detect changed files. From what I could tell, it worked both in the `MGL_EXTERN IB_DESIGNABLE` order and the other way around. I changed the order though so that `IB_DESIGNABLE` now is directly before `@interface`.